asQ/paradiag.py

```diff
@@ -210,7 +228,7 @@ def apply(self, pc, x, y):
 
         # Do the block solves
 
-        print("Warning - for nonlinear we need to update u0")
+        # print("Warning - for nonlinear we need to update u0")
```

Comment:
delete this instead of commenting out

Reply:
shall I also delete the comment above? I.e.
# print("Warning - for nonlinear we need to update u0")
#The above will be done via self.get_appctx(pc)["state"]

Reply:
I mean the second line #The above will ....